templates/terraform/examples/dns_policy_basic.tf.erb

```diff
@@ -20,11 +22,20 @@ resource "google_dns_policy" "<%= ctx[:primary_resource_id] %>" {
 }
 
 resource "google_compute_network" "network-1" {
+  provider = "google-beta"
```

Comment:
Technically this should be the non-beta provider, but it doesn't look like that can be used.

Reply:
This approach is correct! I made the beta provider required for every resource in beta tests / OiCS examples.